src/lib/http/index.ts

```diff
@@ -6,15 +6,16 @@ type HttpConfig = { credentials?: RequestCredentials; headers?: any; addBrackets
 
 const httpConfigParams = ['credentials', 'headers', 'addBracketsToArrayValues'];
 
-export function GET<T = Response>(
-  _requestUrl: string,
-  data: Params | null | undefined | HttpConfig,
-  config?: HttpConfig
-): Promise<T> {
-  const { credentials = 'include', headers = {}, addBracketsToArrayValues = true } = _getHttpConfig(data, config);
-  const requestData = !config && _isConfigObject(data) ? null : data;
+type ParamsOrHttpConfig = Params | null | undefined | HttpConfig;
+
+export function GET<T = Response>(_requestUrl: string, params?: ParamsOrHttpConfig, config?: HttpConfig): Promise<T> {
+  if (_isConfigObject(params)) {
+    config = params;
+    params = {};
+  }
```

Comment:
I rewrote the logic to determine config and params to be a little more straight-forward, with less conditionals.